docs/howto/system_testing.md

```diff
@@ -110,7 +110,7 @@ volumes:
 
 ### Agent service deployer
 
-**NOTE**: To be deprecated soon in favor of creating new Elastic Agents in each test (technical preview yet). These
+**NOTE**: Deprecated in favor of creating [new Elastic Agents in each test](#running-a-system-test). These
```

Comment:
I was thinking to update also the subsection title , but maybe it is referenced somewhere else.